requirements.txt

```diff
@@ -15,6 +15,7 @@ idna==3.4
 importlib-resources==5.12.0
 mistune==3.0.1
 numpy==1.24.3
+openlane==2.0.7
```

Comment:
This ties the openlane version to tt-support-tools - I'm not sure if this is desirable. But I'm happy to merge and see if it causes any issues (we can always override the version by pip installing a different version after installing the requirements).